files/default/templates/hopsworks.slack.tmpl

```diff
@@ -0,0 +1,32 @@
+{{ define "hopsworks.slack.default.title" }}
+     [{{ .Status | toUpper }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}] {{ .CommonLabels.alertname }}
+{{ end }}
+{{ define "hopsworks.slack.default.text" }}
+ {{ with index .Alerts 0 -}}
+     {{- if and .Labels.job .GeneratorURL }}
+        :bar_chart: *<{{ .GeneratorURL }}|{{ .Labels.job }}>*
+     {{- else if and .Labels.featureGroup .GeneratorURL }}
+        :chart_with_upwards_trend: *<{{ .GeneratorURL }}|{{ .Labels.featureGroup }}>*
+     {{- end }}
+ {{ end }}
+ {{ range .Alerts -}}
+   *Alert:* {{ .Annotations.title }}{{ if .Labels.severity }} - `{{ .Labels.severity }}`{{ end }}
+
+   *Summary:* {{ .Annotations.summary }}
+   *description:* {{ .Annotations.description }}
+
+   *Details:*
+      - *Project:* {{ .Labels.project }}
+     {{- if and .Labels.job .Labels.executionId}}
+      - *Job:* {{ .Labels.job }}
+      - *Execution id:* {{ .Labels.executionId }}
+     {{- else if and .Labels.featureGroup .Labels.featureGroupId }}
+      - *Feature group:* {{ .Labels.featureGroup }}
+      - *Feature group id:* {{ .Labels.featureGroupId }}
```

Comment:
Feature group id is useless. The alert should contain feature store name/ feature group name/feature group version.